.github/ISSUE_TEMPLATE/epic-issue.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Epic issue
+about: Create a new epic issue for `Woof` project.
+title: ''
+labels: EPIC, need more discussion
+assignees: ''
+
+---
+
+<!-- This reference to the epic issue makes the navigation process more easily -->
+## This task is a part of the issue:
+ - #_issue
+
+<!-- This references to the issues that block the current issue-->
+## Depends on tasks:
+- #_issue
+- #_issue
+
+<!-- Provides the general description of the issue-->
+## Description
+
+<!-- Provides the general description of the issue-->
+## Description
+
+
+## DoD
+<!-- Describes the requirements from the user's point of view-->
+### User story:
+
+<!-- Describes any restrictions and requirements from the developer's point of view -->
+### Dev requirements and restrictions:
+
+
```

Comment:
Why there are two empty line? Be consistent )
```suggestion
```

Reply:
Reviewed template, kept only one line.